### DISCUSSION TITLE
error in build  openssl-sys

### DISCUSSION THREAD

**Discussion by LuWroblewski** [cargo-lambda/cargo-lambda#625] [Q&A]
i try cargo lamba build --release --arm64 or cargo lamba build --release  and this happens

i using axus, tokio and seaORM

using Windows 11 and powershell

i tryed installing openSSL on wsl and windows but the error continues

 ![image](https://github.com/cargo-lambda/cargo-lambda/assets/92353062/84065eb3-4ed4-4c0b-bd1d-2a6b7008e8d2)
![image](https://github.com/cargo-lambda/cargo-lambda/assets/92353062/dd0ac5ba-9f15-4fbf-b65a-4385d938f1a1)

**Comment by calavera** [comment]
`Ar` is a tool required to link native dependencies, which is missing in your system. I would recommend you however to not use OpenSSL, but Axum and SeaORM have support for Rustls, which should not give you any compilation problem.

**Reply by LuWroblewski** [reply]
i create new folder and testing dependencies

My problem en reqwest

**Reply by calavera** [reply]
Same applies for reqwest. If you don't want to use Rustls, this is another solution: https://github.com/awslabs/aws-lambda-rust-runtime/issues/496#issuecomment-1172970709
